ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/StateProcessor.java

```diff
@@ -202,12 +204,26 @@ private void onAttestation(Attestation attestation) {
       transaction.commit();
       eventBus.post(new StoreDiskUpdateEvent(transaction));
 
-      chainStorageClient.addUnprocessedAttestation(attestation);
     } catch (SlotProcessingException | EpochProcessingException e) {
       STDOUT.log(Level.WARN, "Exception in onAttestation: " + e.toString());
     }
   }
 
+  @Subscribe
+  @SuppressWarnings("unused")
+  private void onBroadcastAttestation(Attestation attestation) {
```

Comment:
Should this be:
```suggestion
  private void onAttestation(Attestation attestation) {
```
?
With the other `onAttestation` method becoming `processAttestation`?

Reply:
the other onAttestation is to be parallel to the on_attestation spec function.

Reply:
switched to onGossipedAttestation after offline discussion.